### PR TITLE
Mention whitespace bug in README

### PR DESCRIPTION
**Pull request by hmelder**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,10 +126,11 @@ Place a breakpoint at the line `NSLog(@"Hello Objective-C");` and run from Visua
 
 ## Status and Known Issues
 
-The toolchain is fully usable on x64. On x86, libdispatch is not available due to a [build error](https://bugs.swift.org/browse/SR-14314).
+- The toolchain is fully usable on x64. On x86, libdispatch is not available due to a [build error](https://bugs.swift.org/browse/SR-14314).
```
Comment (triplef) [inline README.md:129]: Nit: let’s use asterisks for this list to match the lists below.
```suggestion
* The toolchain is fully usable on x64. On x86, libdispatch is not available due to a [build error](https://bugs.swift.org/browse/SR-14314).
```

File: README.md [hunk 2]
```diff
@@ -126,10 +126,11 @@ Place a breakpoint at the line `NSLog(@"Hello Objective-C");` and run from Visua
...
 
-Note that GNUstep support for Windows is not as complete as on Unixes, and some [tests in GNUstep Base](https://github.com/gnustep/libs-base/actions/workflows/main.yml?query=branch%3Amaster) are still failing.
+- Note that GNUstep support for Windows is not as complete as on Unixes, and some [tests in GNUstep Base](https://github.com/gnustep/libs-base/actions/workflows/main.yml?query=branch%3Amaster) are still failing.
```
Comment (triplef) [inline README.md:131]:
```suggestion
* Note that GNUstep support for Windows is not as complete as on Unixes, and some [tests in GNUstep Base](https://github.com/gnustep/libs-base/actions/workflows/main.yml?query=branch%3Amaster) are still failing.
```

File: README.md [hunk 3]
```diff
@@ -126,10 +126,11 @@ Place a breakpoint at the line `NSLog(@"Hello Objective-C");` and run from Visua
...
 
+- The compilation will fail if the Windows username contains whitespace, e.g. 'John Appleseed'.
```
Comment (triplef) [inline README.md:133]:
```suggestion
* The compilation will fail if the Windows username contains whitespace, e.g. 'John Appleseed'.
```

**Review by triplef — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (hmelder) [reply README.md:129]: > Nit: let’s use asterisks for this list to match the lists below.

Ofc

**Pull request merged**